glide.yaml

```diff
@@ -19,7 +17,7 @@ import:
 - package: github.com/tendermint/go-merkle
   version: develop
 - package: github.com/tendermint/go-p2p
-  version: develop
+  version: viper
```

Comment:
> better way

you test it using `glide mirror`, but, right before you send a PR, you change the version to develop, so it could be merged into develop?